modules/mobile/app/workers/mobile/v0/pre_cache_appointments_job.rb

```diff
@@ -7,15 +7,19 @@ class PreCacheAppointmentsJob
 
       sidekiq_options(retry: false)
 
+      class MissingUserError < StandardError; end
```

Comment:
Adding these error classes because I want to see it in sentry if this is happening and at least one of these jobs effectively hides the error from us. We can probably remove this if we find that it never occurs.